tufaceous-lib/src/date.rs

```diff
@@ -3,7 +3,7 @@ use chrono::{DateTime, Duration, Timelike, Utc};
 
 /// Parser for datelike command line arguments. Can accept a duration (e.g.
 /// "1w") or an ISO8601 timestamp.
-pub(crate) fn parse_duration_or_datetime(s: &str) -> Result<DateTime<Utc>> {
+pub fn parse_duration_or_datetime(s: &str) -> Result<DateTime<Utc>> {
```

Comment:
nitpick but since this is only used by the CLI it could go in the other crate. it could even be inlined into `main.rs`